modin/core/io/text/csv_glob_dispatcher.py

```diff
@@ -331,18 +329,9 @@ def file_exists(cls, file_path: str, storage_options=None) -> bool:
         bool
             True if the path is valid.
         """
-        if isinstance(file_path, str) and is_url(file_path):
```

Comment:
What if someone passes in a GitHub raw user content file - with an https irl prefix - would that work?

Reply:
Yeah actually I think it shouldn't work for the glob case. I remember Alexey bringing this up, so I'll revert this back.